Source/Runtime/Clients/Discovery V1/NI Discovery V1 Client.lvlib

```diff
@@ -20,6 +20,9 @@
 		<Item Name="Discovery Client FGV.vi" Type="VI" URL="../Discovery Client FGV.vi">
 			<Property Name="NI.LibItem.Scope" Type="Int">4</Property>
 		</Item>
+		<Item Name="Enumerate Services.vi" Type="VI" URL="../Enumerate Services.vi">
+			<Property Name="NI.LibItem.Scope" Type="Int">4</Property>
```

Comment:
The reason for setting the scope of this VI to community rather than public is that we do not want to expose this API to users. However, in the future, we plan to access this VI from the Measurement Client project. Therefore, in subsequent Measurement Client implementations, this library will be a **Friend** to the Measurement Client related VIs or libraries.